src/libraries/System.Collections/src/Resources/Strings.resx

```diff
@@ -129,9 +129,12 @@
   <data name="ArgumentOutOfRange_BiggerThanCollection" xml:space="preserve">
     <value>Must be less than or equal to the size of the collection.</value>
   </data>
-  <data name="ArgumentOutOfRange_Index" xml:space="preserve">
+  <data name="ArgumentOutOfRange_IndexMustLT" xml:space="preserve">
```

Comment:
Nit, I'd suggest "ArgumentOutOfRange_IndexMustBeLessOrEqual" and "ArgumentOutOfRange_IndexMustBeLess" as more self explanatory (no need to reset validation for this)

Reply:
Thanks! Done.